apps/site/styles/locals.css

```diff
@@ -0,0 +1,3 @@
+html[lang='ko'] {
```

Comment:
```suggestion
/* Comment to explain why, this tile for ko */
html[lang='ko'] {
```

Can you document why, you put theses style for Korean

Reply:
Comments have been added!